.pre-commit-config.yaml

```diff
@@ -306,6 +306,203 @@ repos:
           - --py-version=3.7
           - --platform=linux
 
+      - id: pip-tools-compile
+        alias: compile-linux-py3.8-zmq-requirements
+        name: Linux Py3.8 ZeroMQ Requirements
+        files: ^requirements/((base|zeromq|pytest)\.txt|static/linux\.in)$
+        exclude: ^requirements/static/(centos-6|amzn-2018\.03|lint|cloud|docs|darwin|windows)\.in$
+        args:
+          - -v
+          - --py-version=3.8
+          - --platform=linux
+          - --include=requirements/base.txt
+          - --include=requirements/zeromq.txt
+          - --include=requirements/pytest.txt
+
+      - id: pip-tools-compile
+        alias: compile-darwin-py3.8-zmq-requirements
+        name: Darwin Py3.8 ZeroMQ Requirements
+        files: ^(pkg/osx/(req|req_ext)\.txt|requirements/((base|zeromq|pytest)\.txt|static/darwin\.in))$
+        args:
+          - -v
+          - --py-version=3.8
+          - --platform=darwin
+          - --include=pkg/osx/req.txt
+          - --include=pkg/osx/req_ext.txt
+          - --include=requirements/base.txt
+          - --include=requirements/zeromq.txt
+          - --include=requirements/pytest.txt
+          - --passthrough-line-from-input=^pyobjc(.*)$
+
+#      - id: pip-tools-compile
```

Comment:
@s0undt3ch can we add a comment to this part explaining when it needs to be un-commented? I'm okay with pre-staging in this file, but we need something to remind us that it needs to be re-added.

Either that, or just remove it from this PR and open up a separate one that adds it in.

Reply:
Done